pubspec.yaml

```diff
@@ -42,6 +42,8 @@ dependencies:
   uuid: ^2.2.2
   audioplayers: ^0.16.0
   syncfusion_flutter_gauges: ^18.4.43
+  photo_view: ^0.10.3
+  overlay_screen: ^1.2.0+1
```

Comment:
It looks like this package is not created by a credible source. I usually go for ones that are from verified developers and that are highly rated. Can we use built-in flutter widgets to achieve the same functionality? Or go to a new page to show the image instead of using an overlay?